### ISSUE TITLE
Apollo setup for Gatsby

### ISSUE THREAD

**Issue by mar-bi** [chingu-voyage4/grad.then#235]

**Comment by spences10** [comment]
Apologies for weighing in on all your issues here @mar-bi 

Is Netlify now functioning for teams on a free teir?

**Comment by mar-bi** [comment]
No. Teams there are not for free - https://www.netlify.com/pricing/#teams
In free tier you can invite up to 5 users (better less than 5) to edit the content of the app.

**Comment by spences10** [comment]
🤔

Yeah, and I'm sure GraphCMS is only two team members so not sure how users would submit stories

**Comment by mar-bi** [comment]
It seems, GraphCMS allows mutations of the data from the client.
We can handle the submission of user stories with a form. But it needs some user authentication mechanism.
